corehq/apps/reports/views.py

```diff
@@ -1477,9 +1477,9 @@ def clear_report_caches(request, domain):
 def export_report(request, domain, export_hash, format):
     cache = get_redis_client()
 
-    if cache.exists(export_hash):
```

Comment:
haha this is what i get:
<img width="716" alt="screen shot 2015-08-17 at 10 25 49 am" src="https://cloud.githubusercontent.com/assets/918514/9307107/603dfdd2-44ca-11e5-9077-28a9588ae64a.png">
something must be off. hold on grabbing the versions i'm using